dcos/subcommand.py

```diff
@@ -1,12 +1,17 @@
 from __future__ import print_function
 
+import hashlib
 import json
 import os
+import platform
 import shutil
+import stat
 import subprocess
 import sys
+import zipfile
 from subprocess import PIPE, Popen
 
+import dcos.http
```

Comment:
Which should remove the cycle. Please file a high priority issue and we can fix it right after this PR